starlight/models.py

```diff
@@ -38,12 +43,33 @@ class Competency(models.Model):
     interest = models.IntegerField(choices=INTEREST_CHOICES)
     experience = models.IntegerField(choices=EXPERIENCE_CHOICES)
 
+    def __str__(self):
+        return '%s: Interest: %s, Experience: %s' % (self.skill.name, self.interest, self.experience)
+
 
 class Employee(AbstractBaseUser):
+    def get_short_name(self):
+        pass
+
+    def get_full_name(self):
+        pass
+
     competencies = models.ManyToManyField(Competency, blank=True, related_name='employees')
 
+    full_name = models.CharField(
```

Comment:
IIRC, the `AbstractBaseUser` class should already include fields for `first_name` and `last_name`. So, if we want to return the user's full name, we only need to create a `full_name` property that returns both properties concatenated.